### ISSUE TITLE
{Freeway-Bug}[Inputs.json]: There is no announcement made by talkback or a visual information provided about the range of values the user can enter in the Quantity edit box.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8469]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
Test Environment
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1
Tool: Logitech Bluetooth Keyboard

Repro Steps:

1. Turn on Talkback.
2. Open Adaptive Cards Visualizer Application.
3. Navigate to sample cards using tab key and activate it using Enter key.
4. A dropdown would appear. Activate it using Enter key.
5. Navigate to Inputs using tab key and activate it using Enter key.
6. Navigate to the Visual tab item present at the top. A sample card would appear.
7. Navigate to the Quantity edit box inside the sample card using tab key and activate it using Enter key.
8. Observe the issue.

Actual Result:
There is no announcement made by talkback or a visual information provided about the range of values the user can enter in the Quantity edit box. Only when the user enters a wrong input there is an error message appearing.

Expected Result:
There should be an announcement made by talkback or a visual information provided about the range of values the user can enter in the Quantity edit box. A visual label such as "Enter values from -5 to 5" or a similar announcement should be made by talkback.

User Experience:
All users would get affected if There is no announcement made by talkback or a visual information provided about the range of values the user can enter in the Quantity edit box.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/error-suggestion

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/89897257/232711040-082e2124-8820-49ea-9aeb-01f67ce33cf6.mp4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
need to update the label to include valid range.
